app/presenters/alchemy/base_presenter.rb

```diff
@@ -0,0 +1,30 @@
+module Alchemy
+  class BasePresenter
```

Comment:
Using the name `Alchemy::BasePresenter` for JSON responses is not preferred. Maybe we want to introduce "normal" presenters later, then the name would be taken.

So, I would prefer something like:

```ruby
Alchemy::BaseSerializer
```

and then a `Alchemy::PageTreeSerializer`

Sorry, the `Presenter` name was my idea.